gradle.properties

```diff
@@ -24,7 +24,8 @@ defaultSpringBootCloudVersion=2.0.0.RELEASE
 nodejsVersion=8.11.1
 projectRoot=${project.projectDir}
 
-org.gradle.daemon=false
+org.gradle.daemon=true
+org.gradle.jvmargs=-Xmx2048m
```

Comment:
why increase Gradle memory?

Reply:
Because with addition of the E2E tests it ocassionally fails on memory related issues if we stick with 1GB

Reply:
I suspect that it has something to do with the high level of parallelization and running of integration and E2E tests at the same time.